### DISCUSSION TITLE
App: Transfers at the top of the payee list

### DISCUSSION THREAD

**Discussion by MiriamNZ** [actualbudget/releases#240] [Feature ideas]
Currently the transfer options appear at the bottom of the payee list. Although each option starts "Transfer:" you cant type "t" (or tr or tra) to get to that part of the list. 

The only way to get to the transfers is to scroll down the list. Which is tedious. (I transfer quite often).

Either a side menu from the top
  First item in Payees us Transfer:
  Choosing it brings up a flyout menu of choices

OR: transfers at the top. 
OR: a setting to choose transfers top or bottom.

**Comment by ghost** [comment]
What helps me, is to type the first few letters of the account name I want to transfer to. That reduces the list size and I use arrow keys to navigate down. Maybe that could be a temp solution for you?

**Comment by bpaulien** [comment]
There is a button at the bottom of the payee list called "Make Transfer" Hit that, and the only thing available are accounts available to make a transfer to.

![image](https://user-images.githubusercontent.com/6487363/159484289-a3923516-5733-4614-acc3-b81d5fab6fe8.png)
